roles/subscription-manager/tasks/offline.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: get repo file
+  get_url:
+    url: "http://{{hostvars['wiley-kragle']['ansible_default_ipv4']['address']}}/offline.repo"
```

Comment:
Instead of running the get facts for deploy_node in site.yml to get the IP, could you use the `auto_deploy_node` var we define in the inventory?